src/mono/browser/runtime/loader/config.ts

```diff
@@ -192,6 +192,10 @@ export function normalizeConfig() {
         // ActiveIssue https://github.com/dotnet/runtime/issues/75602
         config.pthreadPoolSize = 7;
     }
+    if (WasmEnableThreads && config.environmentVariables["MONO_SLEEP_ABORT_LIMIT"] === undefined) {
+        config.environmentVariables["MONO_SLEEP_ABORT_LIMIT"] = "5000";
```

Comment:
Is this a production ready value? Why is it 5000? Maybe add a comment

Reply:
Yes, production. I just moved the value from `driver.c` it seems reasonable timeout. Mono default is infinite.